.github/workflows/pull_request_previews.yml

```diff
@@ -0,0 +1,27 @@
+name: pr-preview-sync
+on:
+  schedule:
+    - cron: 0/5 * * * *
```

Comment:
Is `0/5` equivalent to `*/5`? If so, I'd prefer the latter, since it's a form I've seen (and used) more.

Reply:
Yes, it is equivalent. Updated